db/schema.rb

```diff
@@ -124,6 +124,7 @@
     t.string   "channel_mode",     limit: 255
     t.string   "status",           limit: 255
     t.string   "format",           limit: 255
+    t.text     "audio_errors",     limit: 65535
```

Comment:
Wondering if it would be worth standardizing this, since it's always the human-readable version of the `status` field.  Maybe something like `status` and `status_msg`?  Could let us store other human readable things in there down the line, to indicate why `status` is what it is.